pkgs/applications/office/bookletimposer/default.nix

```
@@ -0,0 +1,38 @@
+{ pkgs ? import <nixpkgs> {} }:
+with pkgs;
```

```suggestion
{ lib, fetchFromGitLab, python3, intltool, pandoc, gobject-introspection, wrapGAppsHook, gtk3, distutils_extra, pypdf2, pygobject3 }:
```

I'm not sure if I included everything here; this isn't tested, but this is how dependencies are usually passed in nixpkgs.

---

Right. (for the record `distutils_extra, pypdf2, pygobject3` were actually part of pythonPackages so didn't need to be explicitely mentioned)

This is a forgotten cleanup from when I started with a default.nix outside of nixpkgs.